src/assets/main.css

```
@@ -1,18 +1,18 @@
-@import './base.css';
+@import url('./base.css');
```

Stylelint (17.14.0)

[error] Expected "url('./base.css')" to be "'./base.css'" (import-notation)

(import-notation)